### PR TITLE
Update the SPDX license enums and lists to version 3.9

### PR DESCRIPTION
**Pull request by tsteenbe**
Done by running "./gradlew :spdx-utils:generateSpdxEnums". For details
see https://github.com/spdx/license-list-XML/releases/tag/v3.9.

Some of the 3.9 licenses where included in the 3.8 update (#0ef37d2)
which is why this update only add 3 and not 16 licenses.

Signed-off-by: Thomas Steenbergen <thomas.steenbergen@here.com>

### CODE REVIEW

File: spdx-utils/src/main/resources/exceptions/SHL-2.1 [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+   "Source" form shall mean the preferred form for making modifications, including but not limited to source code, net lists, board layouts, CAD files, documentation source, and configuration files.
+
+   "Work" shall mean the work of authorship or design, whether in Source or Object form, made available under the License, as indicated by a notice relating to Rights that is included in or attached to the work (an example is provided in the Appendix below).
+
+   2. Grant of License. Subject to the terms and conditions of this License, each Contributor hereby grants to You a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable license under the Rights to reproduce, prepare Derivative Works of, make, adapt, repair, publicly display, publicly perform, sublicense, and distribute the Work and such Derivative Works in Source or Object form and do anything in relation to the Work as if the Rights did not exist. APPENDIX
+
+Copyright [yyyy] [name of copyright owner]
+
+SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
```
Comment (sschuberth) [inline spdx-utils/src/main/resources/exceptions/SHL-2.1:35]: Wow, this is interesting, as it uses `WITH` not with an exception, but an addition.

Comment (tsteenbe) [reply spdx-utils/src/main/resources/exceptions/SHL-2.1:35]: See the text is has additions and modifications so it exception plus an addition - have not seen one before ...

Comment (sschuberth) [reply spdx-utils/src/main/resources/exceptions/SHL-2.1:35]: Anyway, this is the proof that it was a good decision from recently to not requite the part of `WITH` to be a known exception.

**Pull request merged**